.pybuild/pythonX.Y_2.7/build/repoman/constants.py

```diff
@@ -22,14 +22,14 @@
 from gi.repository import Gtk
 
 class App:
-    application_id = "com.github.mirkobrombin.ppaextender"
```

Comment:
The application_id need to be: com.github.mirkobrombin.ppaextender or com.github.mirkobrombin.repoman